src/common.js

```diff
@@ -182,6 +183,7 @@ function expand(list, opts) {
   if (!Array.isArray(list)) {
     throw new TypeError('must be an array');
   }
+  opts = opts || config.globOptions;
```

Comment:
Actually, we're not using `opts` anymore (it was added for something, but we refactored the code to not need it). I think it's cleaner to just use `config.globOptions` instead.

I don't think our code should ever need to pass options to `glob.sync()`. We experimented with using `glob.sync('dirName/*', {dot: true})` for `ls -a`, but it's much easier to just us `fs.readDirSync()` instead (it's simple, it's faster, and it actually works correctly for a directory named `dir*With?Glob[chars]/`).